packages/cozy-doctypes/src/Folder.js

```diff
@@ -0,0 +1,105 @@
+const Application = require('./Application')
+const CozyFile = require('./File')
+
+/**
+ * Class representing the folder model.
+ * @extends CozyFile
+ */
+class CozyFolder extends CozyFile {
+  /**
+   * Create a folder with a reference to the given document
+   * @param  {String}  path     Folder path
+   * @param  {Object}  document
```

Comment:
document is a io.cozy.files ?

Reply:
No, I will make the doc clearer about it.